src/Compilers/CSharp/Portable/Syntax/SyntaxFacts.cs

```diff
@@ -152,6 +152,9 @@ public static bool IsInTypeOnlyContext(ExpressionSyntax node)
                     case RefType:
                         return ((RefTypeSyntax)parent).Type == node;
 
+                    case ScopedType:
+                        return ((ScopedTypeSyntax)parent).Type == node;
```

Comment:
Does this change affect any test?

Reply:
> Does this change affect any test?

Yes, there is a direct call to this API in one of the tests